cmd/helm/upgrade.go

```diff
@@ -151,7 +151,15 @@ func (u *upgradeCmd) run() error {
 		// The returned error is a grpc.rpcError that wraps the message from the original error.
 		// So we're stuck doing string matching against the wrapped error, which is nested somewhere
 		// inside of the grpc.rpcError message.
-		_, err := u.client.ReleaseHistory(u.release, helm.WithMaxHistory(1))
+		releaseHistory, err := u.client.ReleaseHistory(u.release, helm.WithMaxHistory(1))
+
+		if err != nil {
+			previousReleaseNamespace := releaseHistory.Releases[0].Namespace
```

Comment:
This causes a panic if running `helm upgrade --install` because the `releaseHistory` doesn't yet exist

Reply:
Oh, yes, fixed it.